package.sh

```diff
@@ -1,15 +1,34 @@
 #!/bin/sh
 
+#Script for create the plugin artifact
+echo "Travis tag: $TRAVIS_TAG"
+
+if [ "$TRAVIS_TAG" = "" ]
+then
+   TRAVIS_TAG='1.0.0'
```

Comment:
Is this a good default? why?

Reply:
Yes, is only for local environmets, in the developer process of the plugin Avoid having to set the environment variable constantly to run pakage.sh in local environments.